lib/migrations/20190806115741_change_student_stage.js

```diff
@@ -0,0 +1,28 @@
+const _ = require("underscore")
```

Comment:
What's the significance of this file?

Reply:
Actually I want to convert All students stage from completedTest to completesedTestWithDetails, that's why I have created this file.